Prog/MFIS/MFIS_2022_01.sas

```diff
@@ -0,0 +1,29 @@
+/**************************************************************************
```

Comment:
@dharvey17 These programs all work great! This is nit-picky, but if you could update the program headers with the program name, author, date create, and issue number, that would be great.